syncmaster/schemas/v1/connections/webdav.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Literal
+
+from pydantic import BaseModel, DirectoryPath, Field, FilePath
+
+from syncmaster.schemas.v1.auth import (
+    CreateBasicAuthSchema,
+    ReadBasicAuthSchema,
+    UpdateBasicAuthSchema,
+)
+from syncmaster.schemas.v1.connection_types import WEBDAV_TYPE
+from syncmaster.schemas.v1.connections.connection_base import (
+    CreateConnectionBaseSchema,
+    ReadConnectionBaseSchema,
+    UpdateConnectionBaseSchema,
+)
+
+
+class CreateWebDAVConnectionDataSchema(BaseModel):
+    host: str
+    port: int | None = None
+    ssl_verify: bool | FilePath | DirectoryPath = True
```

Comment:
As user configures connection via API/UI, passing file/directory path here is not very convenient, as this is path on worker, and users may have no access to worker filesystem. Let's just drop this field, and use `ssl_verify=False` on worker side.